packages/hash/api/src/graphql/resolvers/ontology/entity-type.ts

```diff
@@ -19,18 +19,14 @@ export const createEntityType: ResolverFn<
   MutationCreateEntityTypeArgs
 > = async (_, params, { dataSources, user }) => {
   const { graphApi } = dataSources;
+  /** @todo - should accountId be optional here */
```

Comment:
Why are we allowing entity type to be missing? Just for convenience to not have to pass the ID?

Reply:
Sorry yes that's what I meant, have edited. I don't think I understand though. Why do we allow `accountId` to be _missing_, surely that wouldn't create a type on another account.

I guess I'm saying why don't we just always require an accountId, how convenient is it really to sometimes let it be undefined and to grab it from the current user?

Reply:
I guess it boils down to the frontend not needing to each for the current user, and letting the Graph QL context/session implicitly pass it. I am not sure how inconvenient it would be to require an accout ID for the frontend

Reply:
I'll get rid of the todo because I think we'll revisit anyway as we consolidate the accountId stuff.